gateway/webhook/webhook.go

```diff
@@ -261,6 +263,36 @@ func (webhook *HandleT) batchRequests(sourceDef string, requestQ chan *webhookT)
 	}
 }
 
+func prepareRequestBody(req *http.Request, includeQueryParams bool, sourceType string, sourceListForParsingParams []string) ([]byte, error) {
+	body, err := io.ReadAll(req.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	jsonBody := "{}" // Start with an empty JSON object if no body is present
+	if len(body) > 0 {
+		if !json.Valid(body) {
+			return nil, fmt.Errorf("invalid JSON in request body")
+		}
```

Comment:
Do we even need to check whether it's a valid JSON or not since we are already checking later?

Reply:
removed